source/ada/lsp-ada_handlers.adb

```diff
@@ -2838,8 +2844,20 @@ package body LSP.Ada_Handlers is
       File      : LSP.Types.LSP_String;
       Charset   : Unbounded_String;
       Variables : LSP.Types.LSP_Any;
+      Relocate  : Virtual_File;
```

Comment:
Relocate and Root need to be initialized with No_File

Reply:
Done